core/dbt/tests/tables.py

```diff
@@ -322,6 +322,8 @@ def _ilike(target, value):
 
 
 def get_tables_in_schema(schema):
+    # TODO the get_models_in_schema has a special case for snowflake, do we still need it
```

Comment:
Is this something we intentionally didn't do

Reply:
Yes. We're going to implement things in the adapter in a cleaner way (like overriding methods) instead of these hacky if tests.